SpectraLogic.SpectraRioBrokerClient/Model/RestoreFile.cs

```diff
@@ -40,7 +40,7 @@ public class RestoreFile
         /// The time code range
         /// </summary>
         [JsonProperty(PropertyName = "timeCodeRange", NullValueHandling = NullValueHandling.Ignore)]
-        public TimeCodeRange TimeCodeRange;
+        public string TimeCodeRange;
```

Comment:
Is there a way we can add a custom deserializer for this instead of reverting to string?